src/workloads/workloads.csproj

```diff
@@ -79,8 +79,8 @@
     <!-- Shorten package names to avoid long path issues in Visual Studio -->
     <ItemGroup>
       <!-- These need to be in order - matching rules will try in order. -->
-      <ShortNames Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest">
-        <Replacement>MonoToolChainManifest</Replacement>
+      <ShortNames Include="Microsoft.NET.Workload.Mono.ToolChain.">
```

Comment:
Too much shortening on this line? :-)

Reply:
So we'll end up with mono.manifest or something like that?

Reply:
BTW, I noticed there's already a VSIX package in VS with an ID called "Mono", so as long as we have another qualifier we should be okay.

Reply:
should end up as MonoManifest-7.0.100